lib/bolt/cli.rb

```diff
@@ -922,6 +925,8 @@ def install_project_modules(project, force, resolve)
     # Adds a single module to the project.
     #
     def add_project_module(name, project)
+      require 'bolt/module_installer'
```

Comment:
Is this pulling in something really huge? It seems fine to have this at the top...

Reply:
We have a test for the CLI that running a `show` or `help` command does not load the SSL library, which this does via the `GitSpec` class.

I can rework it so that SSL is only loaded when a request is actually made.